src/error.ts

```diff
@@ -8,6 +8,14 @@ export type AnyError = MongoError | Error;
 /** @internal */
 const kErrorLabels = Symbol('errorLabels');
 
+/** @internal */
+// Abstract the offensive terminology to a constant
+const LEGACY_NOT_PRIMARY_ERROR_MESSAGE_REGEX = /not master/;
```

Comment:
Can we store the strings here as constants, not the regexs?  Then we can export them, and use the constant error messages in the unit tests as well.  

The functions below that use these constants will need to be updated to create a regex from the string 
``` typescript
new RegExp(LEGACY_NOT_PRIMARY_ERROR_MESSAGE)
```

Reply:
I like it!